contracts/deprecated/RegisterAuth.sol

```diff
@@ -6,8 +6,10 @@ struct Authorization {
     bool registerValidator;
 }
 
+/// @notice Deprecated. Notice that if the library is used again,
+/// all the values in AuthData.authorization will still be available.
 library RegisterAuth {
-    uint256 constant private SSV_STORAGE_POSITION = uint256(keccak256("ssv.network.storage.auth")) - 1;
+    uint256 private constant SSV_STORAGE_POSITION = uint256(keccak256("ssv.network.storage.auth")) - 1;
```

Comment:
General question (not related to this code)

We are not deleting the struct for registerAuth right?  I just wanna make sure as this storage is used and we need to make sure we dont use same name again, or anything that can corrupt storge etc ...

Reply:
Yes, thats the reason I moved the library to the _deprecated_ folder instead of deleting it.